app/routers/getjoke.py

```diff
@@ -0,0 +1,10 @@
+from fastapi import APIRouter
+import requests
+
+
+router = APIRouter()
+
+
+@router.get("/getjoke")
+def getjoke():
+    return requests.get(f'http://api.icndb.com/jokes/random').json()
```

Comment:
this should be on internal as it is an internal function, that is needed to be called from a route.

Reply:
Done